pkg/app/run_test.go

```diff
@@ -2565,6 +2565,18 @@ COMMANDS
                                    The name of an existing condition in the
                                    configured endpoint, or leave blank to always
                                    execute
+        --parse-log-keyvals        Parse key-value pairs within the log format
```

Comment:
@kellymclaughlin Is there a reason this doesn't match the `--[no-]parse-log-keyvals` later on in this file?

Reply:
The reason for that difference is that at creation time there is no need to be able to disable this setting (*i.e.* there's no ambiguity at create time) whereas when updating specifying `--no-parse-log-keyvals` explicitly allows one to disable the setting for an endpoint that had previously enabled it.

Reply:
Gotcha. Makes sense. Thanks for clarifying.